images/initials_test.go

```diff
@@ -0,0 +1,27 @@
+package images
```

Comment:
Would it be possible to write a test that outputs the image bytes and compares against a predefined byte slice?

Reply:
Going to merge this PR now, I'll try to add the test later